bundles/org.openhab.binding.philipsair/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,183 @@
+		<label>LED light level</label>
+		<description>Level 100, 75, 50, 25, 0</description>
+		<state pattern="%d" min="0" max="100" step="25" />
+	</channel-type>
+	<channel-type id="cl">
+		<item-type>Switch</item-type>
+		<label>Child lock</label>
+		<description>Child lock enabled/disabled</description>
+	</channel-type>
+
+	<channel-type id="dtrs">
+		<item-type>Number</item-type>
+		<label>Timer left</label>
+		<description>Timer minutes lefts</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="dt">
+		<item-type>Number</item-type>
+		<label>Timer</label>
+		<description>Timer in hours (0-5)</description>
+		<state min="0" max="5" pattern="%d" step="1" />
+	</channel-type>
+
+	<channel-type id="pm25">
+		<item-type>Number:Density</item-type>
+		<label>PM2.5</label>
+		<description>PM 2.5 particles pollution level</description>
+		<state readOnly="true" pattern="%d" min="0" max="5000" />
+	</channel-type>
+	<channel-type id="iaql">
+		<item-type>Number</item-type>
+		<label>IAQL</label>
+		<description>Allergen index</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="aqit">
+		<item-type>Number</item-type>
+		<label>AQIT</label>
+		<description>AQI</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="err">
+		<item-type>String</item-type>
+		<label>Error message</label>
+		<state readOnly="true" pattern="%s">
+			<options>
+				<option value="0">No error</option>
+				<option value="49408">No water</option>
+				<option value="32768">Water tank open</option>
+				<option value="49155">Clean pre-filter</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fltsts0">
+		<item-type>Number</item-type>
+		<label>Pre-filter</label>
+		<description>Pre-filter estimated clean up</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="wicksts">
+		<item-type>Number</item-type>
+		<label>Wick</label>
+		<description>Wick filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="fltsts2">
+		<item-type>Number</item-type>
+		<label>Carbon</label>
+		<description>Active carbon filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="fltsts1">
+		<item-type>Number</item-type>
+		<label>HEPA filter</label>
+		<description>HEPA filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="rh">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Current humidity</description>
+		<category>Humidity</category>
+		<state readOnly="true" pattern="%.1f %unit%" min="0" max="100" />
+	</channel-type>
+	<channel-type id="rhset">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity setpoint</label>
+		<description>Humidity setpoint</description>

```

Comment:
Shouldn't this also be `<category>Humidity</category>`?

Reply:
Yes indeed